Aztec/Classes/NSAttributedString/Conversions/AttributedStringSerializer.swift

```diff
@@ -108,9 +108,19 @@ class AttributedStringSerializer {
         if let representation = implicitRepresentation(for: element, inheriting: childAttributes) {
             content.append(representation)
         } else {
-            for child in element.children {
-                let childContent = serialize(child, inheriting: childAttributes)
-                content.append(childContent)
+            if let imgElement = linkedImageElement(for: element) {
+                let linkText = element.stringValueForAttribute(named: "href") ?? ""
+                let imgAttributes = self.attributes(for: imgElement, inheriting: attributes)
+                if let attachment = imgAttributes[NSAttachmentAttributeName] as? ImageAttachment,
```

Comment:
The specific problem we have at this point is that if this condition fails, we're losing content, since the element has no representation in the attributed string.

Upon inspecting the code in more depth I realized both of these conditions should succeed if we're dealing with an image element.  If they ever failed it would mean we have broken logic somewhere.

The ideal solution would be to reengineer our formatters, but that'd be too big of a project for the scope of the trial.

In order to avoid a silent failure, we could use something like this:

```swift
let attachment = imgAttributes[NSAttachmentAttributeName] as! ImageAttachment
let attributedString = implicitRepresentation(for: imgElement, inheriting: imgAttributes)!
```

PS: I know it can look counter-intuitive but it's basically a decision between silently ignoring an error situation vs letting it "explode" and more noticeable (so we can fix it).

Reply:
Normally I would try to avoid force unwrapping optionals, but I can see how "letting it explode" would be better in this case - to make the failure noticeable. Will make that change momentarily.